### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by kensteph**
In this PR I : 
- Display 4 items per row instead of 3.
- Refactor the displaySeries and the filter functionality to avoid overconsuming the API.
- Improve the performance when filtering the series
- Change the Homepage screenshot.

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -66,34 +66,17 @@ header {
...
-  margin-top: 25px;
+  width: 75vw;
+  margin-top: 15px;
   display: grid;
-  grid-template-columns: auto auto auto;
+  grid-template-columns: 300px 300px 300px 300px;
   gap: 15px;
   padding: 20px;
   overflow-y: scroll;
 }
```
Comment (biruk-bereka) [inline src/css/style.css]: Screenshot (3)
Screenshot (4)


Kindly change the width of the items. I think it is better not to give a fixed width for the items as it creates overflow for small width devices.

**Review by biruk-bereka — CHANGES_REQUESTED** [review]
Hi @kensteph 
Good job on improving the performance when filtering the series. 

But there is some issues you need to work on. 
See the comments below.

**Comment by kensteph** [comment]
Thank you @biruk-bereka ! Done!

**Review by biruk-bereka — APPROVED** [review]
Congratulation!! 🎉🎉🎉 

You can merge it now.

**Pull request merged**